sound/soc/sof/mediatek/mt8195/mt8195.h

```diff
@@ -0,0 +1,154 @@
+/* SPDX-License-Identifier: GPL-2.0 */
+
+/*
+ * Copyright (c) 2021 MediaTek Corporation. All rights reserved.
+ *
+ *  Header file for the mt8195 DSP register definition
+ */
+
+#ifndef __MT8195_H
+#define __MT8195_H
+
+#define DSP_REG_BASE        0x10803000
+#define SCP_CFGREG_BASE  0x10724000
+#define DSP_SYSAO_BASE  0x1080C000
+
+/*****************************************************************************
+ *                  R E G I S T E R       TABLE
+ *****************************************************************************/
+#define DSP_JTAGMUX          (0x0000)
+#define DSP_ALTRESETVEC      (0x0004)
+#define DSP_PDEBUGDATA       (0x0008)
+#define DSP_PDEBUGBUS0       (0x000c)
+#define PDEBUG_ENABLE	 BIT(0)
+#define DSP_PDEBUGBUS1       (0x0010)
+#define DSP_PDEBUGINST       (0x0014)
+#define DSP_PDEBUGLS0STAT    (0x0018)
+#define DSP_PDEBUGLS1STAT    (0x001c)
+#define DSP_PDEBUGPC         (0x0020)
+#define DSP_RESET_SW         (0x0024)
+/*reset sw*/
+#define BRESET_SW	 BIT(0)
+#define DRESET_SW	 BIT(1)
+#define RUNSTALL	 BIT(3)
```

Comment:
These are again very generic definitions that may conflict with other headers in the future. Compiler with catch the definitions so this is a minor note.

Reply:
@kv2019i  I think its my misunderstanding. I think I could modify these definitions with prefix ADSP_. Thanks.